tools/test.py

```diff
@@ -541,7 +541,8 @@ def Run(self):
       result = self.RunCommand(self.GetCommand(), {
         "TEST_SERIAL_ID": "%d" % self.serial_id,
         "TEST_THREAD_ID": "%d" % self.thread_id,
-        "TEST_PARALLEL" : "%d" % self.parallel
+        "TEST_PARALLEL" : "%d" % self.parallel,
+        "NODE_REPL_EXTERNAL_MODULE": "",
```

Comment:
For consistency, handle it like we do with `NODE_PATH` in `Execute()`?

https://github.com/nodejs/node/blob/17ce4a380fa8103725eec572f9575945c5b90949/tools/test.py#L705-L707